pages/hooks/usestate.mdx

```diff
@@ -2,6 +2,30 @@
 title: useState
 ---
 
+import useStateArray from '../../examples/files/react/useStateArray.js'
+
 # useState
 
-Coming soon!
+`const [value, setValue] = useState(initialValue);`
```

Comment:
I think we should start with a quick definition of what `useState` is/does before we go into the API